### ISSUE TITLE
Error: write ECONNABORTED

### ISSUE THREAD

**Issue by abhinandanmadaan** [microsoft/vscode-azureappservice#1206]
After logging into my Azure account from Visual Studio Code, I am trying to deploy my React app to Azure. After the deployment starts, I ran into the following error, and I have no idea what the cause is.
````
4:25:01 PM Demoapp: Creating zip package...
4:25:18 PM Demoapp: Starting deployment...
4:25:19 PM: Error: write ECONNABORTED
````

**Comment by nturinski** [comment]
Hi @abhinandanmadaan 

Is this still recurring?  It could have been just been a temporary outage.  If not, could you send me your React App so that I could try deploying it to investigate?

**Comment by abhinandanmadaan** [comment]
Yes I was able to deploy my app. And it seemed to be a temporary outage, and also a probable reason could be me sitting behind a corporate proxy. It was resolved on its own.

**Comment by nturinski** [comment]
Thanks for confirmation!  Please reopen if you end up having issues again (but hopefully not)
